util.go

```diff
@@ -9,3 +15,21 @@ func sliceContains(group []string, name string) bool {
 
 	return false
 }
+
+// WriteFile writes the buffer to the file.
+func WriteFile(reader io.Reader, path string) error {
+	dirPath := filepath.Dir(path)
+	if err := os.MkdirAll(dirPath, os.ModeDir|os.ModePerm); err != nil {
```

Comment:
I think I asked for this. I think making enclosing directories is going to be useful in this situation, and it would be annoying to have to do it in any of the use cases where we'd be using this functionality.